content/it/docs/concepts/containers/_index.md

```diff
@@ -1,4 +1,30 @@
 ---
-title: "Containers"
+title: Containers
 weight: 40
+description: La tecnologia per distribuire un'applicazione insieme con le dipendenze necessarie per la sua esecuzione.
+content_type: concept
+no_list: true
 ---
+
+<!-- overview -->
+
+Ogni _container_ che viene eseguito è riproducibile; la pratica di includere le dipendenze all'interno di ciascuno _container_ permette di ottenere sempre lo stesso risultato ad ogni esecuzione del medesimo _container_. 
+
+I _Container_ permettono di disaccoppiare le applicazioni dall'infrastruttura del host su cui vengono eseguite. Questo approccio rende più facile il _deployment_ su cloud o sitemi operativi differenti tra loro.
+
+<!-- body -->
+
+## Immagine di container
+L'[immagine di un container](/docs/concepts/containers/images/) e' un pacchetto software pronto per essere eseguito che contiene tutto ciò che serve per eseguire un'applicazione: il codice sorgente e ciascun _runtime_ necessario, librerie applicative e di sistema, e le impostazioni predefinite per ogni configurazione necessaria.
```

Comment:
nit for making the sentence more readable
```suggestion
L'[immagine di un container](/docs/concepts/containers/images/) e' un pacchetto software che contiene tutto ciò che serve per eseguire un'applicazione: il codice sorgente e ciascun _runtime_ necessario, librerie applicative e di sistema, e le impostazioni predefinite per ogni configurazione necessaria.
```

Reply:
done. Waiting for final ok to squash